app/controllers/spree/admin/products_controller_decorator.rb

```diff
@@ -1,13 +1,20 @@
-Spree::Admin::ProductsController.class_eval do
-  before_action :find_stores, only: [:update]
+module Spree
+  module Admin
+    module ProductsControllerDecorator
+      def self.prepended(base)
+        base.before_action :find_stores, only: [:update]
+      end
 
-  private
+      private
 
-  def find_stores
-    store_ids = params[:product][:store_ids]
-    if store_ids.present?
-      params[:product][:store_ids] = store_ids.split(',')
+      def find_stores
```

Comment:
this method should return stores or their ids ? Asking because it's happening during `update` action